release_notes/44-release-notes.adoc

```diff
@@ -563,6 +565,44 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
 +
 This update ensures that the linked collection is retained, avoiding the error message.
 
+[id="resolved-in-version-441_{context}"]
+=== Resolved in version 4.4.1
+
+*Release date*: 22 April 2024
+
+* Fixed an issue that prevented the filtering of newly added or updated policies by category by using the `roxctl` CLI.
+
+[id="resolved-in-version-442_{context}"]
+=== Resolved in version 4.4.2
+
+*Release date*: 15 May 2024
+
+This release provides the following bug fixes:
+
+* Before this update, Collector pods on nodes with 128 or more cores would fail with a `CrashLoopBackOff` status due to issues with how the CO-RE BPF allocated kernel memory. The patch release fixes this issue.
+* This release updates the Scanner baseline vulnerability data to address changes made to the Red{nbsp}Hat security data feeds that were not compatible with earlier data from Scanner's scheduled feed processing. This fixes various issues where vulnerabilities were detected for images containing packages that were incorrectly indicated as affected by a vulnerability.
+* This release fixes a crash and rendering error in the network graph that occurs when Central is running an {product-title-short} release of 4.3.6 or earlier and Sensor is running an {product-title-short} release of 4.4.0 or later.
```

Comment:
Sensor and Central seem like proper nouns, but I couldn't find anywhere else in the docs that they were capitalized without other modifiers. For example, Prism Central. Noting this as something to check.